cmd/support_archive/builder.go

```diff
@@ -35,6 +35,8 @@ const (
 	loadsimFilesFlagName           = "loadsim-files"
```

Comment:
now that I think of it: do we need to update the support archive e2e test as well? Because of the supportability flag?

Reply:
by default `supportability` role is enabled and nothing is changed. However I will update the `support_archive` e2e test to check if EEC diagnostic log files are added to the package.